hs_core/tests/api/native/test_folder_download_zip.py

```diff
@@ -79,5 +79,15 @@ def test_create_temp_zip(self):
         output_path = "zips/rand/multi_sites_formatted_version1.0.refts.json.zip"
 
         self.assertTrue(create_temp_zip(self.res.short_id, input_path,
-                                        output_path, True, sf_zip=True))
+                                        output_path, None, sf_zip=True))
+        self.assertTrue(IrodsStorage().exists(output_path))
```

Comment:
It'd be better to assign ```IrodsStorage()``` to a variable on the top of this test function, then use it in multiple places than instantiating IrodsStorage multiple instances each time when asserting a path exists in iRODS.

Reply:
done